src/main.js

```diff
@@ -122,8 +122,8 @@ if (!window.SplunkRum) {
     if (options.beaconUrl) {
       const completeUrl = options.beaconUrl + (options.rumAuth ? '?auth='+options.rumAuth : '');
       const batchSpanProcessor = new BatchSpanProcessor(new PatchedZipkinExporter(completeUrl), {
-        bufferTimeout: 100, //millis, tradeoff between batching and loss of spans by not sending before page close
-        bufferSize: 20, // spans, traceoff between batching and hitting sendBeacon invididual limits
+        bufferTimeout: 5000, //millis, tradeoff between batching and loss of spans by not sending before page close
```

Comment:
should we discuss in the comment for this line, why the particular value was chosen?

Reply:
I'll make a note that it's still under experimentation...